chassis: 
  chassis_controller:
    type: "chassis_controller/RollerWheel"
    # ChassisBase
    publish_rate: 100
    power:
      effort_coeff: 12.0
      vel_coeff: 0.0048
      power_offset: -3
    twist_angular: 0.5233
    timeout: 0.1
    twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.001]
    # MecanumController
    wheel_base: 0.47
    wheel_track: 0.505
    wheel_radius: 0.07625

    left_front:
      joint: "left_front_wheel_joint"
      pid:
        {
          p: 1.5,
          i: 0.1,
          d: 0.003,
          i_max: 5.0,
          i_min: -5.0,
          antiwindup: true,
          publish_state: true,
        }
    right_front:
      joint: "right_front_wheel_joint"
      pid:
        {
          p: 1.5,
          i: 0.1,
          d: 0.003,
          i_max: 5.0,
          i_min: -5.0,
          antiwindup: true,
          publish_state: true,
        }
    left_back:
      joint: "left_back_wheel_joint"
      pid:
        {
          p: 2.0,
          i: 0.2,
          d: 0.005,
          i_max: 5.0,
          i_min: -5.0,
          antiwindup: true,
          publish_state: true,
        }
    right_back:
      joint: "right_back_wheel_joint"
      pid:
        {
          p: 2.0,
          i: 0.2,
          d: 0.005,
          i_max: 5.0,
          i_min: -5.0,
          antiwindup: true,
          publish_state: true,
        }
